website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -10,6 +10,10 @@ description: |-
 
 Use this resource to create and manage NRQL alert conditions in New Relic.
 
+-> **IMPORTANT!** Version 2.0.0 of the New Relic Terraform Provider introduces some [additional requirements](/docs/providers/newrelic/index.html) for configuring the provider.
+<br><br>
+Before upgrading to version 2.0.0 or later, it is recommended to upgrade to the most recent 1.x version of the provider (version 1.16.0) and ensure that your environment successfully runs `terraform plan` without unexpected changes.
```

Comment:
🤔💭 Should this live here? How do we want to handle this in resource documentation.

Reply:
I like the idea of this showing up in the resources.